src/app/auth/containers/auth/auth.component.ts

```diff
@@ -7,9 +8,32 @@ import { AuthService } from '../../services/auth.service';
   styleUrls: ['./auth.component.scss']
 })
 export class AuthComponent implements OnInit {
+  private login = true;
 
-  constructor() { }
+  constructor(private authService: AuthService) { }
 
-  ngOnInit() { }
+  ngOnInit() {}
 
+  loginToggle() {
+    this.login = !this.login;
+  }
+
+  onLoginRequested(user: UserCredentials) {
+    this.authService.login(user).subscribe((userObj: User) => {
```

Comment:
unsubscribe'ida tuleks ka, ilmselt kõige elegantsem viis oleks kasutada rxjs take operator'it. take(1) peaks andma esimese vastuse API-lt ja siis complete'ima ennast.